.github/workflows/contrib_checks.yml

```diff
@@ -98,91 +98,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      # First do our check with --locked to make sure `Cargo.lock` is up to date
-      - name: Check all features
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: --locked --all-features
-
-      - name: Rustfmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
-
-      - name: Cranky
-        uses: actions-rs/cargo@v1
-        with:
-          command: cranky
-          args: --all-targets --all-features -- --deny warnings
-
-      # Check a few important permutations of the feature flags for our `rerun` library:
-      - name: Check rerun with `--no-default-features``
-        uses: actions-rs/cargo@v1
-        with:
-          command: cranky
-          args: --locked -p rerun --no-default-features
-
-      - name: Check rerun with `--features sdk`
-        uses: actions-rs/cargo@v1
-        with:
-          command: cranky
-          args: --locked -p rerun --no-default-features --features sdk
-
-      - name: Test doc-tests
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --doc --all-features
-
-      - name: cargo doc
-        uses: actions-rs/cargo@v1
-        with:
-          command: doc
-          args: --no-deps --all-features --workspace
-
-      - name: cargo doc --document-private-items
-        uses: actions-rs/cargo@v1
-        with:
-          command: doc
-          args: --document-private-items --no-deps --all-features --workspace
-
-      # Just a normal `cargo test` should always work:
-      - name: cargo test --all-targets
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-
-      # Full test of everything:
-      - name: cargo test --all-targets --all-features
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --all-targets --all-features
-
-  rs-check-wasm:
-    name: Check Rust web build (wasm32 + wasm-bindgen)
-    runs-on: ubuntu-latest-16-cores
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: clippy check re_viewer wasm32
-        shell: bash
-        run: ./scripts/clippy_wasm.sh
-
-      - name: Check re_renderer examples wasm32
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: --locked --target wasm32-unknown-unknown --target-dir target_wasm -p re_renderer --examples
+      - name: install cargo cranky
+        run: cargo install --locked --quiet cargo-cranky
```

Comment:
This is an interesting one and hints at something that's still not converged by our "pixi all the things" strategy. We need a single place that we define our expectations around system environment that isn't covered by pixi, which I think for us involves mostly cargo stuff.

Reply:
yeah all cargo targets etc. aren't covered very well
the wasm target we add in all pixi commands that need it since profiling it showed that it's very fast if the target was already added